subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c

```diff
@@ -253,18 +247,9 @@ static key_type find_key(uint32_t id, platform_key *key)
 	}
 
 	if (usage == USAGE_IAK || usage == USAGE_MKEK || usage == USAGE_MEXT) {
-		key->ikg.domain = domain;
-		switch (usage) {
-		case USAGE_IAK:
-			key->ikg.slot_number = CRACEN_IDENTITY_KEY_SLOT_NUMBER;
-			break;
-		case USAGE_MKEK:
-			key->ikg.slot_number = CRACEN_MKEK_SLOT_NUMBER;
-			break;
-		case USAGE_MEXT:
-			key->ikg.slot_number = CRACEN_MEXT_SLOT_NUMBER;
-			break;
-		}
+		/* IKG keys are populated in cracen_load_keyref and cracen_get_builtin_key
+                     * so we don't need to populate them here.
```

Comment:
Though indentation is off now? (And compliance complains.)

Reply:
yeah, I think that fixed that now. Just an issue with spaces and tabs.